package.json

```diff
@@ -24,7 +24,7 @@
     "router"
   ],
   "scripts": {
-    "test": "jest --coverage --no-cache",
+    "test": "jest --coverage --no-cache --testURL https://hyperapp.js.org",
```

Comment:
I see, very nice and very cool find. I certainly didn't know.

Having said that, I'll probably change this to something else, as I don't want the result of the test to depend on a flag specified on package.json